packages/libraries/cli/src/helpers/print-table.ts

```diff
@@ -0,0 +1,29 @@
+type TableCell = string | number | Date;
+type Table = TableCell[][];
+
+function printCell(cell: TableCell): string {
+  if (cell instanceof Date) {
+    return cell.toISOString();
+  }
+  return String(cell);
+}
+
+function mapTable<T, O>(table: T[][], map: (cell: T, row: number, col: number) => O): O[][] {
+  return table.map((row, r) => row.map((cell, c) => map(cell, r, c)));
+}
+
+export function printTable(table: Table) {
```

Comment:
If there's a package we've used and prefer for printing tables, then I'm happy to replcae this. The few I found had fairly large dependency trees and I wanted to avoid that.

Reply:
We can consider it at some point, for now I think its fine to have it here